Core/scripts/dirac-install.py

```diff
@@ -1044,16 +1418,17 @@ def downloadAndExtractTarball(tarsURL, pkgName, pkgVer, checkHash=True, cache=Fa
   # for member in tf.getmembers():
   #  tf.extract( member )
   # os.chdir(cwd)
-  tarCmd = "tar xzf '%s' -C '%s'" % (tarPath, cliParams.targetPath)
-  os.system(tarCmd)
-  # Delete tar
-  if cache:
-    if not os.path.isdir(cacheDir):
-      os.makedirs(cacheDir)
-    os.rename(tarPath, tarCachePath)
-  else:
-    if tarPath != tarFileCVMFS:
-      os.unlink(tarPath)
+  if not isSource:
+    tarCmd = "tar xzf '%s' -C '%s'" % (tarPath, cliParams.targetPath)
```

Comment:
I would advise to use the `tarfile` module instead of relying on the fact that the `tar` executable is present on the machine.